about.html

```diff
@@ -0,0 +1,178 @@
+        <li id="nav-3"><a href="about.html" class="nav-mobile-links">About</a></li>
+        <li id="nav-3"><a href="#" class="nav-mobile-links">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+  
+
+    <section class="about_main_header_div">
+      
+      <div class="about_header_text_container">
+        <h1 class="about_header_text"> 
+          Together preventing diabetes 2022
+        </h1>
+      </div>
+
+      <div class="about_header_divider_line"></div>
+
+      <p class="about_header_p">
+        A joyful celebration believing in the value of openness and sharing,
+            creating a positive change with people for over 80 countries is taking place in august in Cameroon.
+        We've learned that learning can be enjoyable and interactive as a result of 
+        all of this, and we're delighted to provide the platform that educates people.
+        Are we prepared for the changes?
+      </p>
+
+      <h3 class="contact_us_text">
+        Please contacts us via email  for any further questions 
+        about diseases, heakth  and the conference 2022.
+      </h3>
+      <h3 class="email_address"><a href="#">
+        emmanuellangufor@gmail.com <span><img class="fordward_arrow" src="images/icons/icons_forward.png" alt="forward arrow"></span>
+        </a>
+      </h3>
+        
+    </section>
+
+    <section class="logo_creation_section">
+      <h3 class="logo_creation_section_title">cameroon diabetes conference logo 2022</h3>
+
+      <div class="about_header_divider_line"></div>
+
+      <p class="logo_creation_section_p">
+        The logo was choosen to pass a message on that everything begins with the mind.
+        the conference will take place from the  8.June to
+        9.July. After free gift, health kits and packages will be distributed. 
+         After free consultation with our well trained and experience health experts
+      </p>
+
+      <img class="logo_file" src="images/nutrition-logo.png" alt="brand logo">
+
+    </section>
+
+    <section class="see_the_stem_conference">
+      <h3 class="logo_creation_section_title">See the past DIABETES conference</h3>
+
+      <div class="about_header_divider_line"></div>
+
+      <p class="logo_creation_section_p">
+        take a look at the last two DIABETES conference, which took place in Cameroon
+      </p>
+
+      <div class="past_stem_images">
+        <div class="past_stem_image_1">
+          <div class="past_stem_image_cover">
+            <h3 class="past_stem_Date">
+              2020
+            </h3>
+
+            <p class="past_stem_text">Cameroon Diabetes conference</p>
+
+          </div>
+        </div>
+
+        <div class="past_stem_image_2">
+          <div class="past_stem_image_cover">
+            <h3 class="past_stem_Date">
+              2021
+            </h3>
+
+            <p class="past_stem_text">Changing the mind conference</p>
+
+          </div>
+
+        </div>
```

Comment:
- Please note that the past project section(mobile version ) has some deviations from the original design template. The links in the mobile version should be arranged in a vertical column. Please try to fix this.
Your project:
![past event student](https://user-images.githubusercontent.com/73607512/192114832-5c152601-80d8-4746-9b8a-d15433800b2d.PNG)

Design template:
![past event figma](https://user-images.githubusercontent.com/73607512/192114839-b6819d15-71da-415c-99ec-ade9b4b4f566.PNG)

Reply:
made all the changes thanks